recipes/bitserializer-cpprestjson/all/conanfile.py

```diff
@@ -0,0 +1,39 @@
+from conans import ConanFile, tools
+import os
+
+
+class BitserializerConan(ConanFile):
+    name = "bitserializer-cpprestjson"
+    description = "C++ 17 library for serialization, module for support JSON (implementation based on the CppRestSDK library)"
+    topics = ("serialization", "json")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://bitbucket.org/Pavel_Kisliak/bitserializer"
+    license = "MIT"
+    settings = ("compiler",)
+    no_copy_source = True
+    requires = ("bitserializer/0.9", "cpprestsdk/[>=2.10.16]")
```

Comment:
You should not be using version ranges in CCI

Reply:
Understood, thanks! Fixed here and also will update all other my PRs.